### DISCUSSION TITLE
Documentation of updated* hook needs to be updated for arrays

### DISCUSSION THREAD

**Discussion by Dimonka2** [livewire/livewire#3241] [4. General]
Hi just wanted to point out that `updated*` hook on array property has an additional argument `$index` that is not documented in documentation.
The hook for array looks like:
````php
public property $array = [];
public function updatedArray($value, $index){}
````

**Comment by joshhanley** [comment]
@Dimonka2 if you can, please submit a PR to the docs repo, that would be a great help https://github.com/livewire/docs

Thanks!
